File input output/Write to file/task.md

```diff
@@ -1,22 +1,63 @@
 ## Write to file
 
-If you open a file using `"w"` (write) as the second argument, a new empty file will be created. Note that if another file with the same name exists, it will be deleted. If you want to add some content to an existing file, you should use the `"a"` (append) modifier.  
+As we already mentioned, if you use `"w"` as the second argument in `open()`, the file opens for 
+writing only. A new empty file will be created. If another file with the same name already exists, it 
+will be erased. If you want to add some content to an existing file, you should use the `"a"` 
+(append) modifier.  
   
-Add elements from the `zoo` list to "output.txt".  
+Another file object method, `f.write(string)`, writes the contents of a <i>string</i> to the file, returning the 
+number of characters written.
+
+```python
+f.write('This is a test\n')
+```
+```text
+15
+```
+Other types of objects in text mode need to be converted to a string:
+```python
+value = ('the answer', 42)
+s = str(value)  # convert the tuple to string
+f.write(s)
+```
+```python
+18
+```
+Where the specified text will be inserted in the file depends on the file mode (`"a"` vs `"w"`).
+
+`"a"`:  the text will be inserted at the end of the file.
+
+`"w"`: the file will be emptied before the text will be inserted at the beginning.
+
+If you want to include a symbol, such as a line break, into your string (to start from a new line),
+add it with a `+`:
+```python
+f.write('\n' + 'string,' + ' ' + 'another string')
+```
+This will add a new line and write `"string, another string"`.
+
+In the code editor, add all elements from the `zoo` list to "output.txt" so that the output is added from
+a new line, and the elements are separated by `" and "`. Use the <code>' and '.join(lst)</code> syntax to 
```

Comment:
We have two different kinds of quotes here side by side. We probably should stick to one of them